### ISSUE TITLE
Add command to bump tool version

### ISSUE THREAD

**Issue by erasche** [galaxyproject/planemo#558]
A la https://docs.npmjs.com/cli/version which would allow easier bumping of tools during dev, maybe without the `--pre` versions because that's just excessive

**Comment by jmchilton** [comment]
An awesome idea for sure but I'm bumping to GCC 2018 milestone - feel free to move back to GCC 2017 if you plan to do this in the next week @erasche.

**Comment by peterjc** [comment]
This would at a minimum need to handle plain version tags in the XML, and also those done with XML macros. Tricky.

What about ``README.rst`` file updates? Just out of scope?

**Comment by erasche** [comment]
Sounds good @jmchilton, I'll work on it for next year. 

@peterjc I'm not sure how one would begin about updating the readme, any thoughts here? At least the XML file is structured and easy to edit. I only plan to handle those conforming to semver, and to throw an error for anything else.

**Comment by peterjc** [comment]
I don't think we can automatically update the README files - this should explicitly be out of scope.

Another thing is wrapper scripts (which I believe the IUC avoids where possible) which may or may not have a version tag of their own. Again, too complicated, so out of scope? e.g. https://github.com/peterjc/pico_galaxy/blob/master/tools/effectiveT3/effectiveT3.py

**Comment by erasche** [comment]
Oh, yes, those two are definitely out of scope. To start, this is just to update a single tool xml with a semver version. Once I get that working, I'll try to handle the case of macros (especially things like `@VERSION@.1`). Nothing more than that.

**Comment by simonbray** [comment]
@lorrainealisha75 and I are working on something similar to this at the moment.

ping @bgruening, in case you weren't aware this issue existed
